py4j-python/src/py4j/tests/java_array_test.py

```diff
@@ -68,7 +68,7 @@ def testCharArray(self):
         char_class = self.gateway.jvm.char
         char_array = self.gateway.new_array(char_class, 2)
         char_array[0] = "a"
-        self.assertAlmostEqual(char_array[0], "a")
+        self.assertEqual(char_array[0], "a")
```

Comment:
this should work (assertAlmostEqual) even on Python 2.6, but it was probably a copy-paste error. thanks for fixing it.